src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -169,10 +169,8 @@ protected override void Apply(TransactionOperationContext context, BlittableJson
                     case nameof(ToggleSubscriptionStateCommand):
                         SetValueForTypedDatabaseCommand(context, type, cmd, index, leader);
                         break;
-                    case nameof(PutApiKeyCommand):
-                        PutValue<ApiKeyDefinition>(context, type, cmd, index, leader);
-                        context.Transaction.InnerTransaction.LowLevelTransaction.AfterCommitWhenNewReadTransactionsPrevented +=
-                            () => serverStore.RavenServer.AccessTokenCache.Clear();
+                    case nameof(PutCertificateCommand):
+                        PutValue<CertificateDefinition>(context, type, cmd, index, leader);
```

Comment:
We need to ensure that when we add a certificate to the raft store, we delete it from the local state.
That will avoid confusion down the line.